### PR TITLE
Add PHP 7.4 compatibility

### PR DESCRIPTION
**Pull request by Housik**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         }
     },
     "require": {
-        "php": ">=8.0",
```
Comment (frankdekker) [inline composer.json:15]: Can you add PHP7.4 to `.github/workflows/test.yml`?

**Review by frankdekker — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (frankdekker) [reply composer.json:15]: And update readme.md badge aswell :)

Comment (Housik) [reply composer.json:15]: @frankdekker done - I am doing this for the first time, so excuse me, if I am wrong

Comment (frankdekker) [reply composer.json:15]: @Housik Took a different approach, and added symfony/php80-polyfill instead for the `str_contains` method.
Created v1.1.0 release for this patch. Thanks for the PR.

**Comment by Housik** [comment]
Frank, I did the same, first try was with Symfony polyfills, but it was not working in composer stage, so I reverted it and use `stripos`. I will try again your approach and let you know.

**Comment by Housik** [comment]
@frankdekker Frank, polyfill is not working on php 7.4, can you please return my solution? Polyfill is probably not loaded/active during composer execution.

`
PHP Fatal error:  Uncaught Error: Call to undefined function DR\CodeSnifferBaseline\Plugin\str_contains() in /home/scripts/vfcs-update/logger/vendor/digitalrevolution/php-codesniffer-baseline/src/Plugin/Plugin.php:78
Stack trace:
#0 [internal function]: DR\CodeSnifferBaseline\Plugin\Plugin->onPostInstall()
#1 phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(174): call_user_func()
#2 phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(102): Composer\EventDispatcher\EventDispatcher->doDispatch()
#3 phar:///usr/local/bin/composer/src/Composer/Installer.php(341): Composer\EventDispatcher\EventDispatcher->dispatchScript()
#4 phar:///usr/local/bin/composer/src/Composer/Command/RequireCommand.php(323): Composer\Installer->run()
#5 phar:///usr/local/bin/composer/src/Composer/Command/RequireCommand.php(243): Composer\Command\RequireCommand->doUpdate()
#6 phar:///usr/local/bin/composer/vendor/symfony/console/Command/Command.php(245): Composer\Command\R in /home/mkusy/PhpstormProjects/_vf/scripts/vfcs-update/logger/vendor/digitalrevolution/php-codesniffer-baseline/src/Plugin/Plugin.php on line 78 `

**Comment by Housik** [comment]
It fails in ‘post-install’ part of composer, when you try to install the library, so it is not covered by testing.

**Comment by frankdekker** [comment]
@Housik Updated. It's interesting though because composer itself uses str_contains aswell. It's strange that the polyfill for that doesn't propagate to the plugin. Using strpos now :).

**Pull request merged**